yarn-project/noir-contracts/src/contracts/docs_example_contract/src/main.nr

```diff
@@ -157,21 +158,21 @@ contract DocsExample {
     // docs:start:state_vars-check_return_notes
     #[aztec(private)]
     fn discard_largest_card() {
-        let storage = Storage::init();
+        let storage = Storage::init(Option::some(&mut context), Option::none());
 
         let account = context.msg_sender();
         let options = create_largest_account_card_getter_options(account);
-        let card = actions::get_cards(&mut context, storage.cards, options)[0].unwrap_unchecked();
+        let card = actions::get_cards(storage.cards, options)[0].unwrap_unchecked();
         // highlight-next-line:state_vars-check_return_notes
         assert(card.owner == account);
 
-        actions::remove_card(&mut context, storage.cards, card);
+        actions::remove_card(storage.cards, card);
     }
     // docs:end:state_vars-check_return_notes
 
     // docs:start:functions-UncontrainedFunction
     unconstrained fn get_total_points(account: Field) -> u8 {
-        let storage = Storage::init();
+        let storage = Storage::init(Option::none(), Option::none());
```

Comment:
I was going to suggest adding a method like `Storage::init_view()` that didn't take any params and set both `Option::none`s under the hood. But if we're going to hide this behind a macro, no need to polish it up, right?